### DISCUSSION TITLE
delete of vm data

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#6611] [UTM Help (iOS)]
hi. i need help. i how can delete all data which utm created? I deleted the VM after failing intalation of windows 7, but the data remains and takes up space on my iphone.

i have iphone 15 pro, ios 18 developer beta 6.

**Comment by osy** [comment]
If it's trollstore, you need to delete all the vm from UTM or Files app. If it's from the app store, just deleting UTM SE will do it.
